source/index.ts

```diff
@@ -1,7 +1,6 @@
 import crypto from "crypto";
 
 import axios, {AxiosInstance} from "axios";
-import rax from "retry-axios";
```

Comment:
Dropping `retry-axios` because of an import error with TS: https://github.com/JustinBeckwith/retry-axios/issues/193#issuecomment-1206103924

It also [apparently](https://github.com/JustinBeckwith/retry-axios/issues/205) doesn't work with the latest versions of Axios anyway.

[axios-retry](https://github.com/softonic/axios-retry) looks like an easy enough replacement for a subsequent PR.